cassandra/src/main/resources/logback.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It is probably better to not add a logback.xml here, if it is for console level. I don't think any of our other edge projects do. Unless you are doing this to override some implicit logback or set INFO level? If to set INFO level, we've been doing that with a property in other projects. Since this will likely join docker, might be better to accept the same env variables.

https://github.com/openzipkin/zipkin/blob/master/zipkin-query-service/config/query-dev.scala#L23